source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,256 @@
+.. _Rust: https://www.rust-lang.org/
+
+.. _piggy-bank-preparing:
+
+====================================
+Preparing a testnet node on a server
+====================================
+
+This is the third :ref:`part of a tutorial<piggy-bank>` on smart contract
+development.
+
+So far you have written and tested a piggy bank smart contract in the Rust_ programming
+language.
+
+This part will focus on how you can set up your own testnet node on a server and create a testnet account.
+In more detail, you will create an instance on a server that will host your own testnet node. Secondly, you will create your own testnet account and prepare it so you can use it in the fourth :ref:`part of this tutorial<piggy-bank-deploying>` for deploying and interacting with your piggy bank smart contract.
+
+.. warning::
+
+   The reader is assumed to have basic knowledge of what a blockchain and smart
+   contract is, and some experience with Rust_.
+
+
+Preparation
+===========
+
+Before you start, make sure you create an account on your favourite cloud provider to set up your instance.
+
+You are now ready to select an instance on the cloud provider that matches the following requirements to run a testnet node:
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Hardware (Testnet node)
+     - Recommended
+   * - CPU (Core)
+     - ?
+   * - RAM (Memory)
+     - 8 GB
+   * - Storage
+     - 50 GB
+   * - Operating system
+     - Ubuntu 20.04 x64
+
+.. Note::
+   The above requirements are simple and meant for a testnet node only. Please have a look at the :ref:`requirements <system-requirements-node-mainnet>` for mainnet nodes.
```

Comment:
Is this rendered correctly?

I believe it should be
```suggestion
.. note::

   The above requirements are simple and meant for a testnet node only. Please have a look at the :ref:`requirements <system-requirements-node-mainnet>` for mainnet nodes.
```

Reply:
It is rendered correctly either way. The documentation is not 100% consistent here e.g. the `testing.rst` file uses `note::` as well as `Note::`. Nonetheless, I changed to `note::` and added an extra line after the command because that is similar to how other commands are used in the tutorial.